deployment/sw360baseimage/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright Bosch Software Innovations GmbH, 2016.
+# Part of the SW360 Portal Project.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM debian:jessie
+MAINTAINER admin@sw360.org
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV _update="apt-get update"
+ENV _install="apt-get install -y --no-install-recommends"
+ENV _cleanup="eval apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*"
+ENV _purge="apt-get purge -y --auto-remove"
+
```

Comment:
Here you should use the variables defined above. Otherwise they are useless.
Doing a system upgrade here would be useful as this is your base image and should contain the latest security updates.